packages/server/src/requestPipeline.ts

```diff
@@ -312,8 +312,9 @@ export async function processGraphQLRequest<TContext extends BaseContext>(
       internals.persistedQueries.cache.set(
         queryHash,
         query,
-        // careful with zero here, explicitly checking for `null` or `undefined`
-        ttl != null ? { ttl: internals.persistedQueries?.ttl } : undefined,
+        // Explicitly checking for `undefined` which means "not set" vs 0 or
+        // null which means "no TTL".
```

Comment:
heh, now i'm not sure why it matters that we make ttl:null work if they can pass ttl:0 too, but eh, maintaining existing code...